client/components/section-header/README.md

```diff
@@ -31,9 +30,10 @@ const MyHeader = ( { translate } ) => (
 
 export default localize( MyHeader );
 ```
-## Section Header
-This is the base component and acts as a wrapper for a section's (a list of cards) title and any action buttons that act upon that list (like Bulk Edit or Add New Item).
 
 #### Props
-- `className` - *optional* (string|object) Classes to be added to the rendered component.
-- `label` - *optional* (string) The text to be displayed in the header.
+
+Name | Type | Default | Description
+---- | ---- | ---- | ----
+`className` | `string` | null | (Optional) Classes to be added to the rendered component.
```

Comment:
Instead of marking which ones are optional, we should mark the required ones with an asterisk.

Reply:
Is there an example that I can copy? I'm not sure where we should put the asterisk.

Reply:
you can find an example here: https://github.com/Automattic/wp-calypso/blob/452ee4cacd28b5bbb7fe2d5394fb16271e8be644/docs/component-readme-template.md#props